administrator-manual/en/release_notes.rst

```diff
@@ -35,6 +35,15 @@ Major changes on 2020-11-XX
 
 * ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003
 
+* As default, users belonging to the ``wheel`` group can now access the systems using SSH.
+  This behavior can be changed using the ``AllowLocalGroups`` property.
+  It accepts a comma separated list of group names in ``/etc/groups`` that are granted full SSH and SFTP access.
+
+  To restore previous behavior, run the following commands: ::
```

Comment:
I don't want to pass the idea that the new behavior is applied to existing systems.

```suggestion
  If SSH access for the `wheel` administrative group is not desired, run the following commands: ::
```